fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalBiomeTags.java

```diff
@@ -0,0 +1,170 @@
+	public static final TagKey<Biome> IS_HOT_OVERWORLD = register("is_hot/overworld");
+
+	public static final TagKey<Biome> IS_TEMPERATE = register("is_temperate");
+	public static final TagKey<Biome> IS_TEMPERATE_OVERWORLD = register("is_temperate/overworld");
+
+	public static final TagKey<Biome> IS_COLD = register("is_cold");
+	public static final TagKey<Biome> IS_COLD_OVERWORLD = register("is_cold/overworld");
+
+	public static final TagKey<Biome> IS_WET = register("is_wet");
+	public static final TagKey<Biome> IS_WET_OVERWORLD = register("is_wet/overworld");
+
+	public static final TagKey<Biome> IS_DRY = register("is_dry");
+	public static final TagKey<Biome> IS_DRY_OVERWORLD = register("is_dry/overworld");
+
+	public static final TagKey<Biome> IS_VEGETATION_SPARSE = register("is_sparse_vegetation");
+	public static final TagKey<Biome> IS_VEGETATION_SPARSE_OVERWORLD = register("is_sparse_vegetation/overworld");
+
+	public static final TagKey<Biome> IS_VEGETATION_DENSE = register("is_dense_vegetation");
+	public static final TagKey<Biome> IS_VEGETATION_DENSE_OVERWORLD = register("is_dense_vegetation/overworld");
+
+	public static final TagKey<Biome> IS_CONIFEROUS_TREE = register("is_tree/coniferous");
+	public static final TagKey<Biome> IS_SAVANNA_TREE = register("is_tree/savanna");
+	public static final TagKey<Biome> IS_JUNGLE_TREE = register("is_tree/jungle");
+	public static final TagKey<Biome> IS_DECIDUOUS_TREE = register("is_tree/deciduous");
+
+	public static final TagKey<Biome> IS_VOID = register("is_void");
+	public static final TagKey<Biome> IS_MOUNTAIN_PEAK = register("is_mountain/peak");
+	public static final TagKey<Biome> IS_MOUNTAIN_SLOPE = register("is_mountain/slope");
+	/**
+	 * Biomes consisting primarily of water.
+	 */
+	public static final TagKey<Biome> IS_AQUATIC = register("is_aquatic");
+	/**
+	 * Biomes that lack any natural life or vegetation.
+	 * (Example, land destroyed and sterilized by nuclear weapons)
+	 */
+	public static final TagKey<Biome> IS_WASTELAND = register("is_wasteland");
+	/**
+	 * Biomes whose flora primarily consists of dead or decaying vegetation.
+	 */
+	public static final TagKey<Biome> IS_DEAD = register("is_dead");
+	/**
+	 * Biomes with a large amount of flowers.
+	 */
+	public static final TagKey<Biome> IS_FLORAL = register("is_floral");
+	/**
+	 * For biomes that contains lots of naturally spawned snow.
+	 * For biomes where lot of ice is present, see {@link ConventionalBiomeTags#IS_ICY}.
+	 * Biome with lots of both snow and ice may be in both tags.
+	 */
+	public static final TagKey<Biome> IS_SNOWY = register("is_snowy");
+
+	public static final TagKey<Biome> IS_BADLANDS = register("is_badlands");
+	public static final TagKey<Biome> IS_CAVE = register("is_cave");
+	/**
+	 * Biomes that spawn as part of the large islands outside the center island in The End dimension.
+	 */
+	public static final TagKey<Biome> IS_OUTER_END_ISLAND = register("is_outer_end_island");
```

Comment:
Major nit: Maybe look at the grouping/ordering of theses, IMO it would make sense to put all the end related tags next to each other?

Reply:
Check out the new grouping order in this class. Is it better now?

Reply:
Much better, thanks 👍